tests/DCPS/RtpsRelay/Smoke/run_stress_test1.py

```diff
@@ -0,0 +1,514 @@
+                                 '-UserData', f'relay{self.uid}',
+                                 '-VerticalAddress', relay_config[self.uid]['vertical_address'],
+                                 '-HorizontalAddress', relay_config[self.uid]['horizontal_address'],
+                                 '-MetaDiscoveryAddress', relay_config[self.uid]['meta_discovery_address'],
+                                 '-LogDiscovery', '1',
+                                 '-LogActivity', '1',
+                                 '-LogRelayStatistics', '300',
+                                 '-Lifespan', '300',
+                                 '-Id', f'relay{self.uid}',
+                                 '-PublishRelayStatus', '10',
+                                 '-PublishRelayStatusLiveliness', '30',
+                                 '-RestartDetection', '1',
+                                 '-LogThreadStatus', '0',
+                                 '-AdmissionControlQueueSize', '10',
+                                 '-AdmissionControlQueueDuration', '5',
+                                 '-AdmissionMaxParticipantsRange', '9500-10000',
+                                 '-LogUtilizationChanges', '1',
+                                 '-ORBLogFile', log_path,
+                                 )
+
+class MonitorSched(OneshotSched):
+    def __init__(self, **params):
+        OneshotSched.__init__(self, **params)
+
+    def callback(self):
+        log_path = f'monitor.log'
+        if os.path.exists(log_path):
+            os.remove(log_path)
+        return self.proc_man.run('monitor', f'{DDS_ROOT}/tests/DCPS/RtpsRelay/Smoke/monitor',
+                                 '-DCPSConfigFile', 'monitor.ini',
+                                 '-DCPSDebugLevel', '1',
+                                 '-ORBVerboseLogging', '1',
+                                 '-DCPSTransportDebugLevel', '1',
+                                 '-ORBLogFile', log_path,
+                                 '-DCPSPendingTimeout', '3',
+                                 )
+
+def run_sim(*scheds):
+    event_loop = EventLoop()
+    proc_man = ProcMan()
+    for sched in scheds:
+        sched.ready(event_loop, proc_man)
+    event_loop.run_events()
+
+
+if __name__ == '__main__':
+    argp = argparse.ArgumentParser()
+    argp.add_argument('count', type=int,
+        help='Fixed count of client pairs. If using --ramp-up, ' +
+          'then this is the max count with 0 meaning unlimited.')
+    argp.add_argument('--max-time', type=int, default=0)
+    argp.add_argument('-r', '--ramp-up', type=int, default=0,
+        help='Gradually increase load in these increments')
+    argp.add_argument('--dummy', action='store_true')
+    argp.add_argument('--profile', action='store_true', help='Profile the relay with massif')
+    args = argp.parse_args()
+
+    # Generate keys, certificates, and permissions documents
+    os.system(f'$DDS_ROOT/tools/scripts/dpm.py --init')
```

Comment:
Please use [`subprocess.check_call`](https://docs.python.org/3/library/subprocess.html#subprocess.check_call) instead of `os.system`. Also the `$DDS_ROOT` isn't portable. I would suggest using a f string or turn `DDS_ROOT` into a `Path` with a `/` operator.